src/components/SelectPlace/PlaceComponent.tsx

```diff
@@ -0,0 +1,56 @@
+import { PlaceComponentProps } from '@_types/type';
+import { useNavigate } from 'react-router-dom';
+import { useEffect } from 'react';
+
+export default function PlaceComponent({ places }: PlaceComponentProps) {
+  const navigate = useNavigate();
+
+  useEffect(() => {
+    console.log(places);
+  }, [places]);
```

Comment:
```suggestion
  useEffect(() => {
    console.log(places);
  }, []);
```

해당 useEffect() 훅은 처음 DOM에 마운트 될 때에만 한번 실행되는 목적이라면, 의존성 배열이 empty array로 설정되어도 좋지 않을까요?